build-rector-scoped.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh -l
+
+# local-prefix test
+
+# show errors
+set -e
+
+# script fails if trying to access to an undefined variable
+set -u
+
+
+# functions
+note()
+{
+    MESSAGE=$1;
+
+    printf "\n";
+    echo "[NOTE] $MESSAGE";
+    printf "\n";
+}
+
+
+note "Starts"
+
+# configure here
+NESTED_DIRECTORY="rector-nested"
+SCOPED_DIRECTORY="rector-scoped"
+
+
+
+# ---------------------------
+
+
+note "Coping root files to $NESTED_DIRECTORY directory"
+rsync -av * "$NESTED_DIRECTORY" --quiet
+
+note "Running composer update without dev"
+composer update --no-dev --no-progress --ansi --working-dir "$NESTED_DIRECTORY" # --ignore-platform-req php
+
+# Unpacking PHPStan
+note "Unpacking PHPStan"
+wget https://github.com/box-project/box/releases/download/3.11.0/box.phar -N --no-verbose
+php box.phar extract "$NESTED_DIRECTORY/vendor/phpstan/phpstan/phpstan.phar" "$NESTED_DIRECTORY/vendor/phpstan/phpstan-extracted"
```

Comment:
@ondrejmirtes Hi Ondra, I just want to thank you for the idea of unpacking `phpstan.phar` and using it's code instead of using `phpstan/phpstan-src`. It saved me so much troubles with duplicating your process. 

Now it rector prefixing works well and `scoper.php.inc` is thin as snow layer :)

I never though of that being a possibility until you told me. :bow: